test/RelayClient.test.ts

```diff
@@ -139,99 +135,34 @@ describe('RelayClient', function () {
         } as typeof accountManagerStub;
       });
 
-      it('should return tx request with given relay request data with fees receiver from given hub info', async function () {
-        const expectedRelayRequestData: UserDefinedRelayData = {
-          ...FAKE_ENVELOPING_REQUEST_DATA,
-          feesReceiver: FAKE_HUB_INFO.feesReceiver,
-        };
-        const {
-          relayRequest: { relayData: actualRelayRequestData },
-        } = await relayClient._prepareRelayHttpRequest(
-          FAKE_HUB_INFO,
-          FAKE_RELAY_REQUEST
-        );
-
-        expect(actualRelayRequestData).to.deep.equal(expectedRelayRequestData);
-      });
-
-      it('should return tx request with updated fees receiver from given hub info', async function () {
-        const expectedFeesReceiver = FAKE_HUB_INFO.feesReceiver;
-        const {
-          relayRequest: {
-            relayData: { feesReceiver: actualFeesReceiver },
-          },
-        } = await relayClient._prepareRelayHttpRequest(
-          FAKE_HUB_INFO,
-          FAKE_RELAY_REQUEST
-        );
-
-        expect(actualFeesReceiver).to.equal(expectedFeesReceiver);
-      });
-
       it('should return relay metadata with relay hub address from request', async function () {
         const expectedRelayHubAddress = 'RELAY_HUB_ADDRESS';
         const {
           metadata: { relayHubAddress: actualRelayHubAddress },
-        } = await relayClient._prepareRelayHttpRequest(FAKE_HUB_INFO, {
+        } = await relayClient._prepareHttpRequest({
           ...FAKE_RELAY_REQUEST,
           request: {
-            ...FAKE_RELAY_REQUEST_BODY,
+            ...FAKE_RELAY_REQUEST.request,
             relayHub: expectedRelayHubAddress,
           },
         });
 
         expect(actualRelayHubAddress).to.equal(expectedRelayHubAddress);
       });
 
-      it('should thow error when feesReceiver in hub info is zero address or not defined', async function () {
```

Comment:
Why did we remove this test? are we testing the same thing elsewhere or did we remove that check at all?

Reply:
was removed as thought the fees receiver would have been set elsewhere. not the case anymore so putting it back now.

Reply:
done